docs/getting_started.md

```diff
@@ -0,0 +1,61 @@
+# Getting started
+**(from scratch)**
+
+To use this library for optimizing a given shim system, that system needs to be defined. This definition occurs via two distinct classes:
+
+1. **ShimSpecs( )**
+*System specifications: This is essentially a struct container which defines system parameters concerning amplifier, DAC, etc.*
+
+and
+
+2. **ShimOpt( )**
+*Shim optimization: This class contains a number of methods used to optimize the shim currents for a given field map*
+
+Both are *abstract* classes, meaning they cannot, in themselves, be instantiated as objects. Rather, each specific shim system will require corresponding subclasses that inherit from these two templates. For example, *ShimSpecs_Greg* and *ShimOpt_Greg* refer to the 8-channel "AC/DC" 3T neck coil.
+
+If you would like to register a new shim system, create a new folder Shim_MyNewShim. Within it, place and rename copies of the abstract classes in the folder **Shim_Template/**.
+
+### Shim reference (aka "calibration") maps
+
+Prior to optimizing any shim currents, some experimental data is needed: These *reference maps* relate the longitudinal magnetic field (delta B0) generated by each shim element to the input current (in A).
+
+TODO: Procedure for creating calibration maps. (Also see: Topfer et al. MRM 2016)
+
+Once this experimental gradient-echo data is available, one need needs to provide the paths to the DICOM images and to define the corresponding experimental currents used in ShimOpt_MyNewShim.declarecalibrationparameters()...
+
+TODO cont...
+
+List of available maps:
+- UNF: https://drive.google.com/drive/folders/1DSCTeh9qRCgS55fWLGtsNeYdq8DsRyMg
+
+Download map and copy it under $PATH_REALTIMESHIMMING/data/
```

Comment:
link is dead, and is this procedure still valid?

Reply:
@rtopfer Do you have a new link and is it still valid?

Reply:
gdrive is not sustainable-- we should move to osf as we've started to do-- @po09i can you please initiate? thx (open issue so we track down progress)

Reply:
Opened issue #21